exercises/clock/Clock.csproj

```diff
@@ -15,4 +15,8 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I think this section can be remove.

Reply:
Yup my bad... My Visual Studio always adds these and I forgot to not commit this one.